src/_visibility.css

```diff
@@ -22,8 +22,8 @@
   clip: rect(1px, 1px, 1px, 1px);
 }
 
-@media (--breakpoint-not-small) {
-  .clip-ns {
+@media (--breakpoint-small) {
+  .clip-s {
     position: fixed !important;
     _position: absolute !important;
```

Comment:
The underscore hack died with IE6, there's no reason that I'm aware of to keep this.